front-end/src/domains/game-board/components/game-grid/game-grid.constant.ts

```diff
@@ -0,0 +1,172 @@
+import { Grid, LandType, NodeType, SpaceType } from "./game-grid.types";
+
```

Comment:
Since we have a lot of constant values that can live outside of a component, we can put them in their own `constant` file 👍